exist-core/src/test/xquery/indexing/range.xml

```diff
@@ -78,35 +78,35 @@
         <task>Test empty element content</task>
         <code>doc("/db/test/text1.xml")//a[. = 'A1']</code>
         <xpath xmlns:prof="http://exist-db.org/xquery/profiling">
-            //prof:index[@type = 'range'][@optimization = 2]
+            //prof:index[@type eq 'range'][@optimization = 'Optimized']
```

Comment:
You might consider changing `=` to `eq` - to match the analogous changes in this PR to `optimizer/expressions.xqm`.

Reply:
Good spot!

Reply:
Done.